### PR TITLE
lidarr: init at 0.3.1.471

### PR DESCRIPTION
**Pull request by etu**
###### Motivation for this change
Fork of sonarr (as radarr) but for music instead of series and movies.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: nixos/modules/services/misc/lidarr.nix [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      description = "Lidarr";
+      after = [ "network.target" ];
+      wantedBy = [ "multi-user.target" ];
+      preStart = ''
+        test -d /var/lib/lidarr/ || {
+          echo "Creating lidarr data directory in /var/lib/lidarr/"
+          mkdir -p /var/lib/lidarr/
+        }
+        chown -R lidarr:lidarr /var/lib/lidarr/
+        chmod 0700 /var/lib/lidarr/
```
Comment (symphorien) [inline nixos/modules/services/misc/lidarr.nix]: Alternatively you can use 
```
serviceConfig.StateDirectory=lidarr";
serviceConfig.StateDirectoryMode="0700";
```
see `man systemd.exec`

Comment (bjornfor) [reply nixos/modules/services/misc/lidarr.nix]: Isn't mode 770 better, so that users in the "lidarr" group also have access?

Comment (etu) [reply nixos/modules/services/misc/lidarr.nix]: @symphorien Oh, didn't know that. I mostly copied the package and service from Sonarr. Pushed with 0770 as mode as suggeste by @bjornfor

**Review by worldofpeace — APPROVED** [review]

**Comment by etu** [comment]
Oops, I deleted the branch. Recreated.

**Comment by edwtjo** [comment]
LGTM, rebased and applied in aba95986d2e76b24ad2c45bed2076dda8b1451b3

**Pull request closed without merging**